pkg/api/api.proto

```diff
@@ -109,10 +109,15 @@ message CreateEnvironmentRequest {
 }
 
 service OverviewService {
+  rpc GetConfig (google.protobuf.Empty) returns (GetConfigResponse) {}
   rpc GetOverview (GetOverviewRequest) returns (GetOverviewResponse) {}
   rpc StreamOverview (GetOverviewRequest) returns (stream GetOverviewResponse) {}
 }
 
+message GetConfigResponse {
+  string argocdBaseUrl = 1;
```

Comment:
I would suggest that we make an argoCd sub-message. Then we can group all argoCd related configs.